src/brouwers/kitreviews/models.py

```diff
@@ -95,7 +91,7 @@ class KitReviewVote(models.Model):
     VOTE_TYPES = (
```

Comment:
Up to you, either are fine for me
On Apr 4, 2016 6:16 PM, "Alex Khomenko" <notifications@github.com> wrote:

> In src/brouwers/kitreviews/models.py
> <https://github.com/modelbrouwers/modelbrouwers/pull/47#discussion_r58402837>
> :
>
> > @@ -95,7 +91,7 @@ class KitReviewVote(models.Model):
> >      VOTE_TYPES = (
>
> Or better positive/negative, that makes more sense imo.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/modelbrouwers/modelbrouwers/pull/47/files/c5219b1fa2e9b039ecf102ec03e4ec19127e646c#r58402837>
>